juju/client/connection.py

```diff
@@ -9,13 +9,22 @@
 from http.client import HTTPSConnection
 
 import macaroonbakery.httpbakery as httpbakery
+import macaroonbakery.bakery as bakery
 import websockets
 from juju import utils
 from juju.client import client
 from juju.errors import JujuAPIError, JujuError
 from juju.utils import IdQueue
 
-log = logging.getLogger("websocket")
+log = logging.getLogger('juju.client.connection')
+
+
+class _RedirectError(Exception):
```

Comment:
I know this is really only for internal use, but I'd still prefer to keep all exceptions together in `juju.errors`.  Also, it's not really an error.